api/tests/unit/domain/models/CertificationAssessmentScoreV3_test.js

```diff
@@ -164,10 +164,10 @@ describe('Unit | Domain | Models | CertificationAssessmentScoreV3 ', function ()
   });
 
   describe('status', function () {
-    describe('when a certification has 10 or more answers', function () {
+    describe('when a certification has config.v3Certification.scoring.minimumAnswersRequiredToValidateACertification or more answers', function () {
```

Comment:
"when a certification has at least the minimum number of answers required by the config"